packages/scripts/config/webpack.config.js

```diff
@@ -0,0 +1,44 @@
+/**
+* External dependencies
+*/
+const path = require( 'path' );
+const PnpWebpackPlugin = require( 'pnp-webpack-plugin' );
+
+const NODE_ENV = process.env.NODE_ENV || 'development';
+
+module.exports = {
+	mode: NODE_ENV,
```

Comment:
As soon as we decide how we proceed with #13814, it should be used in this config to ensure we can benefit from shared settings like `externals`.